pkg/admin/graphql/query.go

```diff
@@ -313,5 +314,33 @@ var query = graphql.NewObject(graphql.ObjectConfig{
 				return graphqlutil.NewConnectionFromResult(lazyItems, result)
 			},
 		},
+		"getUsersByStandardAttribute": &graphql.Field{
+			Description: "Get users by standardAttribute, attributeName must be email, phone_number or preferred_username.",
+			Type:        graphql.NewList(nodeUser),
+			Args: relay.NewConnectionArgs(graphql.FieldConfigArgument{
+				"attributeName": &graphql.ArgumentConfig{
+					Type: graphql.String,
```

Comment:
It should be NonNull. The type in the generated schema should be `String!`, not `String`.

Reply:
fixed query args type